core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data.service.spec.ts

```diff
@@ -166,15 +165,11 @@ describe('TrainingDataService', function() {
   });
 
   beforeEach(angular.mock.inject(function($injector, $rootScope) {
-    scope = $rootScope.$new();
-    $httpBackend = $injector.get('$httpBackend');
     siis = $injector.get('StateInteractionIdService');
     ecs = $injector.get('StateEditorService');
-    cls = $injector.get('ChangeListService');
```

Comment:
For example: ChangeListService is not needed here and hence its require is not needed. (Apply it everywhere)

Reply:
Done.